### DISCUSSION TITLE
打开脚本自动就停止了

### DISCUSSION THREAD

**Discussion by wowodewo** [dundunnp/hamibot-auto_xuexiqiangguo#692] [Q&A]
华为nova2plus手机
按照指南已经打开了hamibot的权限，启动脚本马上就自动停止了，查看日志提示：
19:00:12.997/E: Wrapped
javax.net.ssl.SSLHandshakeException:
Chain validation failed
(file///android_ asset/modules/_ _http_ .js#49)
Wrapped javax.net.ssl.SSL .HandshakeException: 
Chain validation failed
at file:/// android_ _asset/modules/_ http__ .js:49:0
at map. _update (6412f668daf5b3653d3bfc58.js:214:0)
at 6412f668daf5b3653d3bfc58.js:238:0

**Comment by MrDarcy8** [comment]
我也是，插眼求解
